taxcalc/simpletaxio.py

```diff
@@ -0,0 +1,648 @@
+"""
+OSPC taxcalc package simple tax input-output class.
+"""
+# CODING-STYLE CHECKS:
+# pep8 --ignore=E402 simpletaxio.py
+# pylint --disable=locally-disabled simpletaxio.py
+
+import os
+import sys
+import json
+import re
+import numpy.core as np
+from numpy.core import int32 as np_int32
```

Comment:
@talumbau,  Again, thanks for the useful suggestion, which has been incorporated in the pull request.